src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -3163,7 +3163,18 @@ class ArnoldGlobals
 			std::sort( outputs->writable().begin(), outputs->writable().end() );
 			IECoreArnold::ParameterAlgo::setParameter( options, "outputs", outputs.get() );
 			IECoreArnold::ParameterAlgo::setParameter( options, "light_path_expressions", lpes.get() );
-
+		
+			// Set the beauty as the output to get frequent interactive updates	
+			unsigned int primaryOutput = 0;
+			for( unsigned int i = 0; i < outputs->readable().size(); i++ )
+			{
+				if( outputs->readable()[i].substr( 0, 5 ) == "RGBA " )
```

Comment:
Prefer `boost::starts_with` - it expresses intent better, and avoids the allocation of a temporary string (not a significant problem here, admittedly).